database.hh

```diff
@@ -845,7 +845,8 @@ public:
      */
     future<std::unordered_set<sstring>> get_sstables_by_partition_key(const sstring& key) const;
 
-    const sstables::sstable_set& get_sstable_set() const;
+    sstables::sstable_set& get_sstable_set() const;
```

Comment:
I should have changed the methods "constness" too. The change was made because we want to be able to capture a version of the returned set (call get() on it) and capturing a version can't be const because the set is counting references to its versions (in the current version the lw_shared_ptr does the counting and the set only knows if the count is not 0, but still, the set makes changes in its contents when the counter changes between 0 and 1)